python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -755,15 +748,24 @@ async def _handle_raise_m1m3(self) -> None:
         self.rem.mtm1m3.evt_detailedState.flush()
 
         # xml 9/10 compatibility
```

Comment:
Is XML 9/10 compatibility still needed?

Reply:
you are right, it is not.. I will strip all these backwards compatibilities with a single individual ticket though.. :)